### DISCUSSION TITLE
1.0.0-beta.14

### DISCUSSION THREAD

**Discussion by mrkai77** [mrkai77/Loop#185] [Announcements]
## 📰 Important News
- 💸 Reminder to support me & Loop by [sponsoring the project](https://github.com/sponsors/MrKai77)!
- 🥳 New icons, featuring Greg Lassale (`@greglassale` on 𝕏)!

## ✨ New Features
- #77 Option to use macOS center instead of screen center (@MrKai77)
- #82  Multi-monitor support! (@MrKai77)
- #151 Possibility to trigger emoji shortcut if just tapped (@MrKai77)
- #154 Larger Window Padding Values (@MrKai77)
- #165 Removed ability to add preset keybinds, in favour of custom cycling keybinds (@MrKai77)
- #170 Improve compatibility with system keybinds (@MrKai77)
- #171 Less intrusive prompt when new icon is unlocked (@MrKai77)
- #184 Add Loop icons by Greg Lassale (@MrKai77)

## 🐛 Bug Fixes
- #148 #149 `WindowDragManager` bug fixes (@MrKai77)
- #116 Repeatedly centering window with padding makes window get smaller (@MrKai77)

---

**Full Changelog**: https://github.com/MrKai77/Loop/compare/1.0.0-beta.13...1.0.0-beta.14

<hr /><em>This discussion was created from the release <a href='https://github.com/MrKai77/Loop/releases/tag/1.0.0-beta.14'>1.0.0-beta.14</a>.</em>

**Comment by JaceThings** [comment]
"Option to use macOS center instead of screen center" is a problem i didn't have but i can't live without it now

**Comment by Harut12345** [comment]
Hello,
I would like to know if you can include this feature of Windows 10 that automatically displays other windows and places them in the remaining space when clicked on.
Like in this video:
https://youtu.be/F2SczvZkpik

**Reply by JaceThings** [reply]
Make a an issue
